program/rust/src/lib.rs

```diff
@@ -1,11 +1,19 @@
+//this is auto generated by build_bpf.sh
+mod c_oracle_header;
+
 #[link(name = "cpyth")]
 extern "C" {
     fn c_entrypoint(input: *mut u8) -> u64;
 }
 
 #[no_mangle]
 pub extern "C" fn entrypoint(input: *mut u8) -> u64 {
-    unsafe{
-        return c_entrypoint(input);
+    let c_ret_val = unsafe{c_entrypoint(input)};
+    if c_ret_val == c_oracle_header::SUCCESSFULLY_UPDATED_AGGREGATE{
+        return 0;
```

Comment:
i take it there's no way to reference SUCCESS here right? maybe leave a comment that this is SUCCESS?

Reply:
The only way is by generating bindings for solana_sdk.h (which I encountered a lot of errors doing, and is probably an overkill), or copying their enums over to the file we are generating bingens for (this sounds not robust to Solana updates unless we copy them in a programatic way, or add to some to do list that people have to do when bumping solana version).